src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -631,9 +654,19 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
         );
         if (target) {
             if (currentHandleIdRef.current !== OATUntargetedRelationshipName) {
+                const relationshipAmount = elements.filter(
+                    (element) =>
+                        element.source &&
+                        element.source === currentNodeIdRef.current
+                ).length;
```

Comment:
Will this always generate the right "next" value? I.e. if I add relationship_0 and relationship_1, then remove relationship_0, won't the next relationship end up at relationship_1 again?